assets/scss/templates/_header.scss

```diff
@@ -13,7 +13,6 @@
   display: flex;
   font-family: $dp-font-family;
   height: auto;
-  overflow: hidden;
```

Comment:
Does it not generate any unwanted behavior?

I mean, this CSS was there for a reason, right?

Reply:
For an unknown reason. This prevents the lateral menu for showing, I would say we need to test this thoroughly.

Reply:
Please check this case:
![image](https://user-images.githubusercontent.com/2439363/60096796-46927480-9728-11e9-8c12-f018decddcaf.png)